src/components/MapView/Analyser/index.tsx

```diff
@@ -404,6 +404,15 @@ function Analyser({ extent, classes }: AnalyserProps) {
     }
   };
 
+  const getTranslatedColumn = (
+    analysis: BaselineLayerResult | PolygonAnalysisResult,
+  ) => {
+    return getAnalysisTableColumns(
+      analysis,
+      !isEnglishLanguageSelected(i18n),
+    ).map(col => ({ ...col, label: t(col.label) }));
+  };
```

Comment:
With all these modifications I would instead make `getAnalysisTableColums` a custom hook (useAnalysisTableColumns) and hide away all of this extra logic there.

Reply:
done